### PR TITLE
feat: make it serve on your machine

### PR DESCRIPTION
**Pull request by mopp**
## Issues

- https://github.com/vim-jp/vimconf.org/issues/113

## Overview

This PR adds the instruction to serve this site on your local machine.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
 gem 'github-pages', '>= 160'
+
+gem "webrick", "~> 1.8"
```
Comment (mopp) [inline Gemfile:5]: jekyll requires webrick.

```console
❯ bundle exec jekyll serve
Configuration file: /home/mopp/repos/vimconf.org/_config.yml
            Source: /home/mopp/repos/vimconf.org
       Destination: /home/mopp/repos/vimconf.org/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 0.603 seconds.
 Auto-regeneration: enabled for '/home/mopp/repos/vimconf.org'
bundler: failed to load command: jekyll (/home/mopp/.anyenv/envs/rbenv/versions/3.1.2/bin/jekyll)
/home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve/servlet.rb:3:in `require': cannot load such file -- webrick (LoadError)
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve/servlet.rb:3:in `<top (required)>'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:184:in `require_relative'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:184:in `setup'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:102:in `process'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:93:in `block in start'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:93:in `each'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:93:in `start'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/jekyll-3.9.3/exe/jekyll:15:in `<top (required)>'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/bin/jekyll:25:in `load'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/bin/jekyll:25:in `<top (required)>'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/cli/exec.rb:58:in `load'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/cli/exec.rb:58:in `kernel_load'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/cli/exec.rb:23:in `run'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/cli.rb:492:in `exec'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/cli.rb:34:in `dispatch'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/cli.rb:28:in `start'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/exe/bundle:37:in `block in <top (required)>'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.4.19/exe/bundle:29:in `<top (required)>'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/bin/bundle:25:in `load'
	from /home/mopp/.anyenv/envs/rbenv/versions/3.1.2/bin/bundle:25:in `<main>'
```

**Review by aiya000 — APPROVED** [review]
環境試してないけどLGTM!

File: README.md [hunk 2]
```diff
@@ -5,3 +5,20 @@
...
+```console
+# Setup Ruby
+# You can use rbenv.
+
+cd vimconf.org
+bundle install
+bundle exec jekyll serve
+```
+
+TODO: Add container version to keep the environment crean.
```
Comment (mattn) [inline README.md:24]:
```suggestion
TODO: Add container version to keep the environment clean.
```

Comment (mopp) [reply README.md:24]: oops!

**Review by mattn — APPROVED** [review]

**Comment by mattn** [comment]
Thank you

**Pull request merged**